data/redirects.yml

```diff
@@ -600,6 +600,8 @@
   - /feedback/runtime/
 "https://docker.qualtrics.com/jfe/form/SV_3OGeFazYFGv4Zj8":
   - /feedback/desktop-build/
+"https://docker.qualtrics.com/jfe/form/1234":
```

Comment:
keeping in draft as we need the final Qualtrics survey link (currently dummy one).